src/Bridges/Constraint/norm_spec_nuc_to_psd.jl

```diff
@@ -0,0 +1,169 @@
+"""
+    NormSpectralBridge{T}
+
+The `NormSpectralCone` is representable with a PSD constraint, since
+``t \\ge \\sigma_1(X)`` if and only if ``[tI X; X^\\top tI] \\succ 0``.
+"""
+struct NormSpectralBridge{T, F, G} <: AbstractBridge
+    row_dim::Int # row dimension of X
+    column_dim::Int # column dimension of X
+    psd_index::CI{F, MOI.PositiveSemidefiniteConeTriangle}
+end
+function bridge_constraint(::Type{NormSpectralBridge{T, F, G}}, model::MOI.ModelLike, f::MOI.AbstractVectorFunction, s::MOI.NormSpectralCone) where {T, F, G}
+    f_scalars = MOIU.eachscalar(f)
+    t = f_scalars[1]
+    row_dim = s.row_dim
+    column_dim = s.column_dim
+    @assert row_dim <= column_dim # TODO informative error if not
```

Comment:
What's the issue if it's not the case

Reply:
assuming rows <= cols allows us to keep the bridge code simpler. it's what Hypatia does, so it also avoids having to mess around with swapping indices in the MOI wrapper there. It's easy enough at the JuMP level to use ``X'`` instead of ``X`` if matrix ``X`` has more rows than columns. I'd propose to keep this restriction and if someone wants to remove it in future then they can without causing breakage.